src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformation.java

```diff
@@ -48,6 +58,32 @@ public ReleaseCreationInformation(
         this.pegoutCreationRskTxHash = pegoutCreationRskTxHash;
         this.pegoutBtcTx = pegoutBtcTx;
         this.pegoutConfirmationRskTxHash = pegoutConfirmationRskTxHash;
+
+        this.utxoOutpointValues = decodeUtxoOutpointValues(transactionReceipt);
+    }
+
+    private List<Coin> decodeUtxoOutpointValues(TransactionReceipt transactionReceipt) {
```

Comment:
Do you need to pass the TransactionReceipt as param? It's already part of the class

Reply:
Good point!

Reply:
Done

Reply:
Fixed